diff -Naur ipu6-drivers-71e0c690292586fc92d4b92b35a40e3ef6a87641.old/dkms.conf ipu6-drivers-71e0c690292586fc92d4b92b35a40e3ef6a87641/dkms.conf
--- ipu6-drivers-71e0c690292586fc92d4b92b35a40e3ef6a87641.old/dkms.conf	2024-05-06 13:20:53.832570173 +0200
+++ ipu6-drivers-71e0c690292586fc92d4b92b35a40e3ef6a87641/dkms.conf	2024-05-06 13:21:17.968644737 +0200
@@ -1,5 +1,5 @@
 PACKAGE_NAME=ipu6-drivers
-PACKAGE_VERSION=0.0.0
+PACKAGE_VERSION=0
 
 MAKE="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir"
 CLEAN="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir clean"
@@ -35,83 +35,83 @@
 
 BUILT_MODULE_NAME[0]="intel-ipu6"
 BUILT_MODULE_LOCATION[0]="drivers/media/pci/intel/ipu6"
-DEST_MODULE_LOCATION[0]="/updates"
+DEST_MODULE_LOCATION[0]="/extra"
 
 BUILT_MODULE_NAME[1]="intel-ipu6-isys"
 BUILT_MODULE_LOCATION[1]="drivers/media/pci/intel/ipu6"
-DEST_MODULE_LOCATION[1]="/updates"
+DEST_MODULE_LOCATION[1]="/extra"
 
 BUILT_MODULE_NAME[2]="intel-ipu6-psys"
 BUILT_MODULE_LOCATION[2]="drivers/media/pci/intel/ipu6"
-DEST_MODULE_LOCATION[2]="/updates"
+DEST_MODULE_LOCATION[2]="/extra"
 
 BUILT_MODULE_NAME[3]="hm11b1"
 BUILT_MODULE_LOCATION[3]="drivers/media/i2c"
-DEST_MODULE_LOCATION[3]="/updates"
+DEST_MODULE_LOCATION[3]="/extra"
 
 BUILT_MODULE_NAME[4]="ov01a1s"
 BUILT_MODULE_LOCATION[4]="drivers/media/i2c"
-DEST_MODULE_LOCATION[4]="/updates"
+DEST_MODULE_LOCATION[4]="/extra"
 
 BUILT_MODULE_NAME[5]="ov01a10"
 BUILT_MODULE_LOCATION[5]="drivers/media/i2c"
-DEST_MODULE_LOCATION[5]="/updates"
+DEST_MODULE_LOCATION[5]="/extra"
 
 BUILT_MODULE_NAME[6]="ov02c10"
 BUILT_MODULE_LOCATION[6]="drivers/media/i2c"
-DEST_MODULE_LOCATION[6]="/updates"
+DEST_MODULE_LOCATION[6]="/extra"
 
 BUILT_MODULE_NAME[7]="ov02e10"
 BUILT_MODULE_LOCATION[7]="drivers/media/i2c"
-DEST_MODULE_LOCATION[7]="/updates"
+DEST_MODULE_LOCATION[7]="/extra"
 
 BUILT_MODULE_NAME[8]="hm2170"
 BUILT_MODULE_LOCATION[8]="drivers/media/i2c"
-DEST_MODULE_LOCATION[8]="/updates"
+DEST_MODULE_LOCATION[8]="/extra"
 
 BUILT_MODULE_NAME[9]="hm2172"
 BUILT_MODULE_LOCATION[9]="drivers/media/i2c"
-DEST_MODULE_LOCATION[9]="/updates"
+DEST_MODULE_LOCATION[9]="/extra"
 
 BUILT_MODULE_NAME[10]="hi556"
 BUILT_MODULE_LOCATION[10]="drivers/media/i2c"
-DEST_MODULE_LOCATION[10]="/updates"
+DEST_MODULE_LOCATION[10]="/extra"
 
 
 if version_lt ${KERNEL_VERSION} ${KV_OV2740}; then
     BUILT_MODULE_NAME[11]="ov2740"
     BUILT_MODULE_LOCATION[11]="drivers/media/i2c"
-    DEST_MODULE_LOCATION[11]="/updates"
+    DEST_MODULE_LOCATION[11]="/extra"
 fi
 
 if version_lt ${KERNEL_VERSION} ${KV_IVSC}; then
     BUILT_MODULE_NAME[12]="ljca"
-    DEST_MODULE_LOCATION[12]="/updates"
+    DEST_MODULE_LOCATION[12]="/extra"
 
     BUILT_MODULE_NAME[13]="spi-ljca"
-    DEST_MODULE_LOCATION[13]="/updates"
+    DEST_MODULE_LOCATION[13]="/extra"
 
     BUILT_MODULE_NAME[14]="gpio-ljca"
-    DEST_MODULE_LOCATION[14]="/updates"
+    DEST_MODULE_LOCATION[14]="/extra"
 
     BUILT_MODULE_NAME[15]="i2c-ljca"
-    DEST_MODULE_LOCATION[15]="/updates"
+    DEST_MODULE_LOCATION[15]="/extra"
 
     BUILT_MODULE_NAME[16]="mei-vsc"
-    DEST_MODULE_LOCATION[16]="/updates"
+    DEST_MODULE_LOCATION[16]="/extra"
 
     BUILT_MODULE_NAME[17]="intel_vsc"
-    DEST_MODULE_LOCATION[17]="/updates"
+    DEST_MODULE_LOCATION[17]="/extra"
 
     BUILT_MODULE_NAME[18]="mei_csi"
-    DEST_MODULE_LOCATION[18]="/updates"
+    DEST_MODULE_LOCATION[18]="/extra"
 
     BUILT_MODULE_NAME[19]="mei_ace"
-    DEST_MODULE_LOCATION[19]="/updates"
+    DEST_MODULE_LOCATION[19]="/extra"
 
     BUILT_MODULE_NAME[20]="mei_pse"
-    DEST_MODULE_LOCATION[20]="/updates"
+    DEST_MODULE_LOCATION[20]="/extra"
 
     BUILT_MODULE_NAME[21]="mei_ace_debug"
-    DEST_MODULE_LOCATION[21]="/updates"
+    DEST_MODULE_LOCATION[21]="/extra"
 fi
