diff -Naur ipu6-drivers-c6f2924c47e68837404cc7c09f0d75bc2581b87e.old/dkms.conf ipu6-drivers-c6f2924c47e68837404cc7c09f0d75bc2581b87e/dkms.conf
--- ipu6-drivers-c6f2924c47e68837404cc7c09f0d75bc2581b87e.old/dkms.conf	2024-10-27 21:39:28.101480200 +0100
+++ ipu6-drivers-c6f2924c47e68837404cc7c09f0d75bc2581b87e/dkms.conf	2024-10-27 21:39:44.061572947 +0100
@@ -38,47 +38,47 @@
 
 BUILT_MODULE_NAME[0]="hm11b1"
 BUILT_MODULE_LOCATION[0]="drivers/media/i2c"
-DEST_MODULE_LOCATION[0]="/updates"
+DEST_MODULE_LOCATION[0]="/extra"
 
 BUILT_MODULE_NAME[1]="ov01a1s"
 BUILT_MODULE_LOCATION[1]="drivers/media/i2c"
-DEST_MODULE_LOCATION[1]="/updates"
+DEST_MODULE_LOCATION[1]="/extra"
 
 BUILT_MODULE_NAME[2]="ov01a10"
 BUILT_MODULE_LOCATION[2]="drivers/media/i2c"
-DEST_MODULE_LOCATION[2]="/updates"
+DEST_MODULE_LOCATION[2]="/extra"
 
 BUILT_MODULE_NAME[3]="ov02c10"
 BUILT_MODULE_LOCATION[3]="drivers/media/i2c"
-DEST_MODULE_LOCATION[3]="/updates"
+DEST_MODULE_LOCATION[3]="/extra"
 
 BUILT_MODULE_NAME[4]="ov02e10"
 BUILT_MODULE_LOCATION[4]="drivers/media/i2c"
-DEST_MODULE_LOCATION[4]="/updates"
+DEST_MODULE_LOCATION[4]="/extra"
 
 BUILT_MODULE_NAME[5]="hm2170"
 BUILT_MODULE_LOCATION[5]="drivers/media/i2c"
-DEST_MODULE_LOCATION[5]="/updates"
+DEST_MODULE_LOCATION[5]="/extra"
 
 BUILT_MODULE_NAME[6]="hm2172"
 BUILT_MODULE_LOCATION[6]="drivers/media/i2c"
-DEST_MODULE_LOCATION[6]="/updates"
+DEST_MODULE_LOCATION[6]="/extra"
 
 BUILT_MODULE_NAME[7]="hi556"
 BUILT_MODULE_LOCATION[7]="drivers/media/i2c"
-DEST_MODULE_LOCATION[7]="/updates"
+DEST_MODULE_LOCATION[7]="/extra"
 
 if ! version_lt ${KERNEL_VERSION} ${KV_OV05C10}; then
     BUILD_EXCLUSIVE_CONFIG="CONFIG_VIDEO_V4L2_I2C CONFIG_V4L2_CCI_I2C"
     BUILT_MODULE_NAME[8]="ov05c10"
     BUILT_MODULE_LOCATION[8]="drivers/media/i2c"
-    DEST_MODULE_LOCATION[8]="/updates"
+    DEST_MODULE_LOCATION[8]="/extra"
 fi
 
 if version_lt ${KERNEL_VERSION} ${KV_OV2740}; then
     BUILT_MODULE_NAME[8]="ov2740"
     BUILT_MODULE_LOCATION[8]="drivers/media/i2c"
-    DEST_MODULE_LOCATION[8]="/updates"
+    DEST_MODULE_LOCATION[8]="/extra"
 fi
 
 if ! version_lt ${KERNEL_VERSION} ${KV_IPU6_ISYS}; then
@@ -87,46 +87,46 @@
 
 BUILT_MODULE_NAME[9]="intel-ipu6-psys"
 BUILT_MODULE_LOCATION[9]="drivers/media/pci/intel/ipu6/psys"
-DEST_MODULE_LOCATION[9]="/updates"
+DEST_MODULE_LOCATION[9]="/extra"
 
 if version_lt ${KERNEL_VERSION} ${KV_IPU6_ISYS}; then
     BUILT_MODULE_NAME[10]="intel-ipu6"
     BUILT_MODULE_LOCATION[10]="drivers/media/pci/intel/ipu6"
-    DEST_MODULE_LOCATION[10]="/updates"
+    DEST_MODULE_LOCATION[10]="/extra"
 
     BUILT_MODULE_NAME[11]="intel-ipu6-isys"
     BUILT_MODULE_LOCATION[11]="drivers/media/pci/intel/ipu6"
-    DEST_MODULE_LOCATION[11]="/updates"
+    DEST_MODULE_LOCATION[11]="/extra"
 fi
 
 if version_lt ${KERNEL_VERSION} ${KV_IVSC}; then
     BUILT_MODULE_NAME[12]="ljca"
-    DEST_MODULE_LOCATION[12]="/updates"
+    DEST_MODULE_LOCATION[12]="/extra"
 
     BUILT_MODULE_NAME[13]="spi-ljca"
-    DEST_MODULE_LOCATION[13]="/updates"
+    DEST_MODULE_LOCATION[13]="/extra"
 
     BUILT_MODULE_NAME[14]="gpio-ljca"
-    DEST_MODULE_LOCATION[14]="/updates"
+    DEST_MODULE_LOCATION[14]="/extra"
 
     BUILT_MODULE_NAME[15]="i2c-ljca"
-    DEST_MODULE_LOCATION[15]="/updates"
+    DEST_MODULE_LOCATION[15]="/extra"
 
     BUILT_MODULE_NAME[16]="mei-vsc"
-    DEST_MODULE_LOCATION[16]="/updates"
+    DEST_MODULE_LOCATION[16]="/extra"
 
     BUILT_MODULE_NAME[17]="intel_vsc"
-    DEST_MODULE_LOCATION[17]="/updates"
+    DEST_MODULE_LOCATION[17]="/extra"
 
     BUILT_MODULE_NAME[18]="mei_csi"
-    DEST_MODULE_LOCATION[18]="/updates"
+    DEST_MODULE_LOCATION[18]="/extra"
 
     BUILT_MODULE_NAME[19]="mei_ace"
-    DEST_MODULE_LOCATION[19]="/updates"
+    DEST_MODULE_LOCATION[19]="/extra"
 
     BUILT_MODULE_NAME[20]="mei_pse"
-    DEST_MODULE_LOCATION[20]="/updates"
+    DEST_MODULE_LOCATION[20]="/extra"
 
     BUILT_MODULE_NAME[21]="mei_ace_debug"
-    DEST_MODULE_LOCATION[21]="/updates"
+    DEST_MODULE_LOCATION[21]="/extra"
 fi
